lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -33,16 +34,33 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
     final formatter = NumberFormat.decimalPattern('en');
 
     final cfdTradingService = context.watch<CfdTradingChangeNotifier>();
+    final cfdOffersChangeNotifier = context.watch<CfdOfferChangeNotifier>();
+
     Order order = widget.order!;
+    final offer = cfdOffersChangeNotifier.offer;
 
     final openPrice = formatter.format(order.openPrice);
     final liquidationPrice = formatter.format(order.liquidationPrice);
-    final estimatedFees = order.estimatedFees.display(currency: Currency.btc, sign: true).value;
-    final margin = order.margin.display(currency: Currency.btc).value;
-    final unrealizedPL = order.pl.display(currency: Currency.btc, sign: true).value;
+    final estimatedFees = order.estimatedFees.display(sign: true).value;
+    final margin = order.margin.display().value;
+    final unrealizedPL = order.pl.display(sign: true).value;
     final expiry = DateFormat('dd.MM.yy-kk:mm').format(order.expiry);
     final quantity = order.quantity.toString();
     final tradingPair = order.tradingPair.name.toUpperCase();
+    final leverage = order.leverage;
+
+    var takerAmount = 0;
+    var makerAmount = 0;
+
+    if (offer != null) {
+      final marginAsSats = order.margin.asSats;
+      // TODO: this should be stored in the order
+      final totalValueAsSats = marginAsSats + marginAsSats * leverage;
+      final takerPnlAsSats = order.pl.asSats;
```

Comment:
🤔  Shouldn't that pl be calculated? I think this is still mocked data isn't it?

Reply:
I think so, I was hoping this is done as part of #201

Reply:
fair enough, I am going to add it then 😝

Reply:
https://github.com/itchysats/10101/pull/255